.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+force_alphabetical_sort = True
```

Comment:
(it seems to work on my machine too) also nice work having the nerves to figure that out 😄 👍